content/en/service_management/events/correlation/_index.md

```diff
@@ -11,6 +11,11 @@ further_reading:
 algolia:
   tags: ["event correlation", "event grouping", "correlation pattern"]
 ---
+{{% site-region region="gov" %}}
+<div class="alert alert-warning">
+Case Management is not available in the {{< region-param key=dd_datacenter code="true" >}} site.
```

Comment:
Event Correlation is not available you mean?

Reply:
updated; thanks for catching!